MVCFinal/src/main/java/Vista/PanelPoblacion.java

```diff
@@ -0,0 +1,809 @@
+package Vista;
+
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.SystemColor;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.sql.SQLException;
+import java.text.NumberFormat;
+import javax.swing.JButton;
+import javax.swing.JFormattedTextField;
+import javax.swing.JLabel;
+import javax.swing.JList;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTextField;
+import javax.swing.SwingConstants;
+import javax.swing.text.NumberFormatter;
+import Controlador.ControladorPanelPoblacion;
+import javax.swing.JComboBox;
+import javax.swing.DefaultComboBoxModel;
+import javax.swing.DefaultListModel;
+import javax.swing.JFormattedTextField.AbstractFormatter;
+
+public class PanelPoblacion extends JPanel {
```

Comment:
809 lineas...

Reply:
Tenemos toda la funcionalidad del programa en un único panel, vamos a intentar reducir de alguna manera las líneas.

Reply:
Habeis pensado en algo así? https://stackoverflow.com/questions/23855974/multiple-panels-in-jframe

Reply:
No entiendo muy bien como aplicar esa metodología a nuestro código, hemos reducido algunas líneas.

Reply:
Lo vamos a dar por bueno...